### DISCUSSION TITLE
What metadata field to use for narrator name?

### DISCUSSION THREAD

**Discussion by JustinCEO** [openaudible/openaudible#685] [Q&A] [answered]
Suppose you want to import an m4b file that isn't from audible. Suppose also that the metadata on the file is missing the narrator. My understanding is that OpenAudible can't edit the metadata fields currently. Suppose I'm using a different software application that can edit the metadata on an m4b. If I'm using this tool to edit metadata, where should I put the narrator name so that OpenAudible recognizes it?

At first I thought maybe it was the "composer" field, based on the metadata of the files that OpenAudible creates from its download of my audible library, but that doesn't seem to work (i.e. if I edit the name of the narrator into an m4b, and import it into audible, the field is still blank in openaudible).

**Comment by openaudible** [comment, marked as answer]
"composer" is what we use. Its possible we are failing to read that value it when we do an import, but I would have to run a test. 
We are working on tag editing! But importing automatically would be best.
